diff --git a/projects/rdbms-framework/build.gradle b/projects/rdbms-framework/build.gradle
--- a/projects/rdbms-framework/build.gradle
+++ b/projects/rdbms-framework/build.gradle
@@ -38,6 +38,5 @@
     implementation 'org.slf4j:slf4j-api:1.7.36'
     
-    antlr 'org.antlr:antlr4:4.11.1'
     
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.1'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.1'
